template/README.md

```diff
@@ -0,0 +1,28 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I'd say it was bootstrapped with `create-contentful-app`, mentioning CRA in "Learn More" is enough